.safety-policy.yml

```diff
@@ -199,6 +199,10 @@ security:
             reason: Fixed cryptography version 41.0.6 requires Python>=3.7 and is used there
         62817:
             reason: Fixed prompt-toolkit version 3.0.13 requires Python>=3.6 and is used there
+        63687:
+            reason: GitPython, Python 2.7, 3.6. fixed version 3.1.35 requires Python>=3.7
+        64227:
+            reason: jinga2 <= version 3.1.3 fixes vulnerability.
```

Comment:
Typo "jinga2"

Second, I suggest to actually upgrade the packages where possible, e.g. Jinja2 3.1.3 for py>=3.7

Reply:
Package update for Python >= 3.7 and safety issue kept.

Reply:
This turned out to be more complex than thought since each time I fixed something, another issue popped up.  That include new requirements for the safety policies, more issues in requirements and finally a bad release of the build package that kept hanging around long after the developers has supposedly pulled that version.